src/template-tags/tickets.php

```diff
@@ -123,6 +123,11 @@ function tribe_tickets_display_count_and_ticket_button() {
 		// get an array for ticket and rsvp counts
 		$types = Tribe__Tickets__Tickets::get_ticket_counts( $event_id );
 
+		// if no rsvp or tickets return
+		if ( ! $types ) {
```

Comment:
If there are no tickets it returns an empty array.  both of those indexes will be arrays as well and not equal 0 them selves. 

$types['rsvp']    = array(
				'count'     => 0,
				'stock'     => 0,
				'unlimited' => 0,
				'available' => 0,
			);